### PR TITLE
Feature/ng bootstrap components

### PR DESCRIPTION
**Pull request by akhilarv**

### CODE REVIEW

File: src/app/_components/birthday/birthday-detail/birthday-detail.component.ts [hunk 1]
```diff
@@ -32,15 +32,19 @@ export class BirthdayDetailComponent implements OnInit {
   }
   
   getBirthdayDetailById(id) {
-    this.birthdayDetail = this.birthdayService.getBirthdayById(parseInt(id));
-    console.log(this.birthdayDetail);  
+    this.birthdayService.getBirthdayById(parseInt(id))
+    .subscribe(res => {
+        this.birthdayDetail  = res['data'];//console.log(this.birthdayDetail);
```
Comment (anoopnadesan) [inline src/app/_components/birthday/birthday-detail/birthday-detail.component.ts]: remove comment

File: src/app/_components/ng-bootstrap/ng-bootstrap.component.html [hunk 2]
```diff
@@ -0,0 +1,185 @@
+<style>
+  button{
```
Comment (anoopnadesan) [inline src/app/_components/ng-bootstrap/ng-bootstrap.component.html]: Remove styles

File: src/app/_services/birthday.service.ts [hunk 3]
```diff
@@ -1,77 +1,38 @@
...
 
 import { Birthday } from '../_models/birthday';
 
-@Injectable()
+@Injectable({
+    providedIn: 'root'
+ })
 export class BirthdayService {
 
+    public apiUrl = 'http://localhost/gitpro/angular-crud-jwt/src/public';
```
Comment (anoopnadesan) [inline src/app/_services/birthday.service.ts]: apiBaseUrl

File: src/app/_services/birthday.service.ts [hunk 4]
```diff
@@ -1,77 +1,38 @@
...
   
-    updateBirthdayById(birthday): Observable<Birthday> {
-        if (birthday.id === 0) {             
-            var birthdayArray = JSON.parse(localStorage.getItem('birthdayData'));
-            var birthdayid = birthdayArray.length;
-                birthday.id = ++birthdayid;
-                birthdayArray.push(birthday);
-            localStorage.setItem('birthdayData', JSON.stringify(birthdayArray));
+    updateBirthdayById(birthday) {
+        let serData =  JSON.stringify(birthday);
```
Comment (anoopnadesan) [inline src/app/_services/birthday.service.ts]: change var name to birthdaysSerialized or something meaningful

File: src/app/_services/birthday.service.ts [hunk 5]
```diff
@@ -1,77 +1,38 @@
...
     deleteBirthdayDetail(id) {
-       var birthdayArray = JSON.parse(localStorage.getItem('birthdayData'));
-        for (var i in birthdayArray) {
-            if (birthdayArray[i].id === id) {
-                birthdayArray.splice(i, 1);
-                localStorage.setItem('birthdayData', JSON.stringify(birthdayArray));  
-            }
-        }    
-    };    
+        console.log(id);
```
Comment (anoopnadesan) [inline src/app/_services/birthday.service.ts]: pls remove console.log

File: src/app/_services/todo.service.ts [hunk 6]
```diff
@@ -1,84 +1,38 @@
...
-                    email: 'jacob_thornton@gmail.com'
-                }
-            ];
-            localStorage.setItem('localData', JSON.stringify(todoArrayData));
-            this.todos = JSON.parse(localStorage.getItem('localData'));
-            console.log('First');
-        }
-        return of(this.todos);
+
+       return this.http.get('http://localhost/gitpro/angular-crud-jwt/src/public/todo');
```
Comment (anoopnadesan) [inline src/app/_services/todo.service.ts]: do the same\

**Review by akhilarv — COMMENTED** [review]
please review

**Review by anoopnadesan — APPROVED** [review]

**Pull request merged**